src/app/fdctl/config_parse.c

```diff
@@ -407,7 +408,7 @@ fdctl_cfg_validate( config_t * cfg ) {
   CFG_HAS_NON_ZERO( snapshots.incremental_snapshot_interval_slots );
 
   CFG_HAS_NON_EMPTY( layout.affinity );
-  CFG_HAS_NON_EMPTY( layout.solana_labs_affinity );
+  CFG_HAS_NON_EMPTY( layout.agave_affinity );
```

Comment:
What's the behavior here if they specify solana_labs_affinity but not agave_affinity? It will produce an empty value error right? I think we want to allow that, so operator doesn't have to update their config.

Reply:
https://github.com/firedancer-io/firedancer/pull/2498/files#diff-4063f4668e77e6c1002f6c54c0ef5d9fd728a2f3474fd2132abc7a770c033466R664 addresses that. So if `agave_affinity` is non-empty, we will use that. If it is empty (which by default it is), it'll take the value of `solana_labs_affinity`, so operators won't have to update their config.